quobyte-csi-driver/values.yaml

```diff
@@ -1,58 +1,92 @@
 # Used to generate CSI deployment files specific to given k8s version
 k8sVersion: "1.20.2"
 quobyte:
+  namespace: kube-system
+  storageClasses:
```

Comment:
This is user configuration/cluster specific and does not belong in CSI driver. Please use your custom chart to have these templates.